_el/partner.md

```diff
@@ -1,323 +1,141 @@
-> what we want. We want an institution that will pursue our social goals
-> in a way that is in line with the individualism at the heart of the
-> liberal ideal, aiming to solve social problems by seeking to change
-> the hearts, minds, and capacities of individual students. Another way
-> of putting this is that we want an institution through which we can
-> express our social goals without violating the principle of individual
-> choice that lies at the center of the social structure, even if this
-> comes at the cost of failing to achieve these goals. So education can
-> serve as a point of civic pride, a showplace for our ideals, and a
-> medium for engaging in uplifting but ultimately inconsequential
-> disputes about alternative visions of the good life. At the same time,
-> it can also serve as a convenient whipping boy that we can blame for
-> its failure to achieve our highest aspirations for ourselves as a
-> society.
+Αντίθετα με ότι συμβαίνει στο μεγαλύτερο μέρος αυτού του βιβλίου, αυτό το κεφάλαιο στηρίζεται περισσότερο σε πράγματα που έχω δει παρά σε πράγματα που έχω κάνει. Οι περισσότερες απόπειρες μου να φέρω αλλαγές σε μεγάλους οργανισμούς υπήρξαν μη-παραγωγικές (κάτι που αποτέλεσε έναν από τους λόγους για τους οποίους παράτησα μία θέση σε πανεπιστήμιο για να ξεκινήσω ξανά το [Software Carpentry](http://carpentries.org) το 2010). Αν και οι συνεισφορές είναι καλοδεχούμενες για οποιοδήποτε κομμάτι αυτού του βιβλίου, θα ήμουν ιδιαίτερα ευγνώμων να ακούσω τι έχετε να πείτε για τα ζητήματα που συζητάμε παρακάτω.
 
-How do efforts to teach computational thinking and digital citizenship
-in schools fit into this framework?
+## Συνεργασία Με Σχολεία {#s:partner-schools}
 
-### Institutional Adoption (whole class/15)
 
-Re-read the list of motivations to adopt new practices given in
-[s:partner-schools](#SECTION). Which of these apply to you and your
-colleagues? Which are irrelevant to your context? Which do you emphasize
-if and when you interact with people working in formal educational
-institutions?
+Όλοι φοβούνται το άγνωστο και την έκθεση του εαυτού τους. Ως αποτέλεσμα, οι περισσότεροι προτιμούν να αποτύχουν παρά να αλλάξουν. Για παράδειγμα, η Lauren Herckis εξέτασε [γιατί οι καθηγητές πανεπιστημίων δεν υιοθετούν βελτιωμένες τεχνικές διδασκαλίες](https://www.insidehighered.com/news/2017/07/06/anthropologist-studies-why-professors-dont-adopt-innovative-teaching-methods). Βρήκε πως ο κύριος λόγος είναι ο φόβος να φανούν ανόητοι μπροστά στους φοιτητές τους. 
+Δευτερεύοντες λόγοι αφορούσαν στο γεγονός πως τα αναπόφευκτα τραντάγματα από τις αλλαγές στις μεθόδους διδασκαλίας θα επηρέαζαν τις αξιολογήσεις των μαθημάτων, και στην επιθυμία να συνεχίζουν να μιμούνται τους δασκάλους που είχαν εμπνεύσει τους ίδιους. Δεν έχει νόημα να συζητήσουμε αν αυτά τα ζητήματα είναι «πραγματικά» ή όχι: οι καθηγητές πιστεύουν πως είναι, οπότε κάθε σχέδιο για συνεργασία μαζί τους πρέπει να τα λαμβάνει υπόψη.
```

Comment:
αφορούσαν στο γεγονός -> αφορούσαν το γεγονός
τραντάγματα  -> μέσα σε ελληνικά εισαγωγικά
θα επηρέαζαν τις αξιολογήσεις των μαθημάτων, και στην  -> θα είχαν αρνητική επίδραση στις αξιολογήσεις των μαθημάτων τους, καθώς και στην

Reply:
@fpsom Thanx, corrected in the new commit, εκτός από το
"αφορούν στο γεγονός" για το οποίο προτιμώ τον εμπρόθετο προσδιορισμό (αν και είναι και οι δύο αποδεκτές https://sarantakos.wordpress.com/2018/06/22/afora/
https://www.asxetos.gr/articles/glossa-ellhnikh/to-rima-aforo-se.html
)